src/status_im2/contexts/communities/overview/utils.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.communities.overview.utils
+  (:require [clojure.string :as string]))
+
+(defn join-existing-users-string
+  [user-list]
+  (let [users-count (count user-list)
+        first-two   (->>
+                     user-list
+                     (take 2)
+                     (map #(first (string/split (:full-name %) #" "))))]
```

Comment:
Since you're already using the thread last macro, you can *pipeline* a bit more for a little bit extra clarity :)

```clojure
(->> user-list
     (take 2)
     (map #(string/split (:full-name %) #" "))
     (map first))
```

It's also more common to see the first argument aligned with the thread macro. I actually thought zprint would format like this automatically.